user_guide_src/source/cli/cli_generators.rst

```diff
@@ -223,6 +223,27 @@ Options:
     future releases. Please use ``make:migration`` as replacement. Also, please use ``make:migration --session``
     to use instead of the deprecated ``session:migration``.
 
+make:validation
```

Comment:
Put this section just below section of make:migration and just above the first note in line 208.

Also, title overline is too short. You can check the check warning for this.

Reply:
resolved